Sources/unstringify/Internal/template.swift

```diff
@@ -1,22 +1,33 @@
 import Foundation
+import UnstringifyFramework
 
 // swiftlint:disable:next function_body_length
-func template(keys: [String], formattedKeys: [FormatKey], richKeys: [String], richFormattedKeys: [FormatKey]) -> String {
-    let file: String =
+func template(keys: [String], formattedKeys: [FormatKey], richKeys: [String], richFormattedKeys: [FormatKey], templatePath: String? = nil) -> String {
+    do {
+        if let templatePath = templatePath {
+            var templateContent = try readFile(at: templatePath)
+            templateContent = replaceVariables(content: templateContent, keys: keys, formattedKeys: formattedKeys, richKeys: richKeys, richFormattedKeys: richFormattedKeys)
+            return templateContent
+        }
+    } catch {
+        
```

Comment:
Unhandled error fallback in catch clause. Should fail gracefully when readFile fails.

Reply:
Done!

Reply:
I disagree, just don't catch this error if you cannot recover from it

Reply:
for that you'll need to add `throws` to the method signature